Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity.yml

```diff
@@ -0,0 +1,862 @@
+    - contextPath: MicrosoftCloudAppSecurity.Alerts.comment
+      description: The comment relating to the alert.
+      type: String
+    - contextPath: MicrosoftCloudAppSecurity.Alerts.resolveTime
+      description: The resolve time of the alert.
+      type: Date
+  - arguments:
+    - default: false
+      description: |-
+        Multiple alerts matching the specified filters.
+        Alert_id should appear similar to - "1234567890abcdefg".
+        Mandatory, unless you use a custom filter.
+      isArray: true
+      name: alert_ids
+      required: false
+      secret: false
+    - default: false
+      description: 'Filter that you create yourself. If you use this custom filter,
+        other filters will not work. The filter template can be found at: https://docs.microsoft.com/en-us/cloud-app-security/api-activities#filters.'
+      isArray: false
+      name: custom_filter
+      required: false
+      secret: false
+    - default: false
+      description: Comment describing why the alerts were dismissed.
+      isArray: false
+      name: comment
+      required: false
+      secret: false
+    deprecated: false
+    description: Command to resolve multiple alerts matching the specified filters.
+    execution: false
+    name: microsoft-cas-alert-resolve-bulk
+  - arguments:
+    - default: false
+      defaultValue: '50'
+      description: Skips the specified number of records.
+      isArray: false
+      name: skip
+      required: false
+      secret: false
+    - default: false
+      defaultValue: '50'
+      description: Maximum number of records returned to the user.
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    - default: false
+      description: Filter activities originating from the specified IP address.
+      isArray: false
+      name: ip
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: 'Filter activities with the specified subnet categories. Valid
+        values are: Corporate, Administrative, Risky, VPN, Cloud_provider, Other.'
+      isArray: false
+      name: ip_category
+      predefined:
+      - Corporate
+      - Administrative
+      - Risky
+      - VPN
+      - Cloud_provider
+      - Other
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: 'Filter activities by the actions taken on them. Valid values are:
+        block, proxy, BypassProxy, encrypt, decrypt, verified, encryptionFailed, protect,
+        verify, null.'
+      isArray: false
+      name: taken_action
+      predefined:
+      - block
+      - proxy
+      - BypassProxy
+      - encrypt
+      - decrypt
+      - verified
+      - encryptionFailed
+      - protect
+      - verify
+      - 'null'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: 'Filter all activities by source type. Valid values are: Access_control,
```

Comment:
Why specify the valid values if they are predefined?
Also applies to other places in the code